desktop/core/src/desktop/views.py

```diff
@@ -61,20 +56,26 @@
 from desktop.lib.paths import get_desktop_root
 from desktop.lib.thread_util import dump_traceback
 from desktop.lib.view_util import is_ajax
-from desktop.log.access import access_log_level, access_warn, AccessInfo
-from desktop.log import set_all_debug as _set_all_debug, reset_all_debug as _reset_all_debug, \
-                        get_all_debug as _get_all_debug, DEFAULT_LOG_DIR
-from desktop.models import Settings, hue_version, _get_apps, UserPreferences
+from desktop.log import (
+  DEFAULT_LOG_DIR,
+  get_all_debug as _get_all_debug,
+  reset_all_debug as _reset_all_debug,
+  set_all_debug as _set_all_debug,
+)
+from desktop.log.access import AccessInfo, access_log_level, access_warn
+from desktop.models import Settings, UserPreferences, _get_apps, hue_version
 from libsaml.conf import REQUIRED_GROUPS, REQUIRED_GROUPS_ATTRIBUTE
-from useradmin.models import get_profile
-from useradmin.models import User
+from useradmin.models import User, get_profile
+
+standard_library.install_aliases()
 
 if sys.version_info[0] > 2:
   from io import StringIO as string_io
+
   from django.utils.translation import gettext as _
 else:
-  from StringIO import StringIO as string_io
   from django.utils.translation import ugettext as _
+  from StringIO import StringIO as string_io
```

Comment:
Good to change this section to:
```python
from django.utils.translation import gettext as _
from io import StringIO as string_io
```

Reply:
I think you are asking to exchange positions within the `if` like:
```
if sys.version_info[0] > 2:
  from django.utils.translation import gettext as _
  from io import StringIO as string_io

else:
  from django.utils.translation import ugettext as _
  from StringIO import StringIO as string_io
```
But as soon as I do this, ruff is complaining. I am not sure if you are asking about changing these lines only?

Reply:
No we can get rid of the if condition because now we only support py3 so try removing the full if condition and only add the required imports normally.

If ruff is then complaining, then simply fix linting issues again using its auto-fix feature.